examples/project_path.py

```diff
@@ -0,0 +1,8 @@
+import questionary
```

Comment:
I tested this example and a <kbd>backspace</kbd> did not redo the autocomplete. Not sure if this is a bug or intended.

Reply:
that's the same for my file auto completion (e.g. when using `cd`). a tab after that backspace will reopen the completion

Reply:
Ok in that case it sounds like it's working as intended!